nucypher/characters/chaotic.py

```diff
@@ -132,15 +133,16 @@ class Felix(Character, NucypherTokenActor):
     TEMPLATE_NAME = 'felix.html'
 
     # Intervals
-    DISTRIBUTION_INTERVAL = 60*60   # seconds (60*60=1Hr)
-    DISBURSEMENT_INTERVAL = 24      # (24) hours
+    DISTRIBUTION_INTERVAL = 60  # seconds (60*60=1Hr)
+    DISBURSEMENT_INTERVAL = 24 * 365  # only distribute tokens to the same address once each year.
```

Comment:
At minimum - This needs a TODO to reduce DB reads.

Reply:
it's sqlite.  it's like reading jpg once every minute.